src/game/engine/foot.cpp

```diff
@@ -243,8 +268,24 @@ RadioMessageType FootClass::Receive_Message(RadioClass *radio, RadioMessageType
     return RadioMessageType();
 }
 
+/**
+ *
+ *
+ * @address 0x004C33C4
+ */
 void FootClass::Sell_Back(int a1)
 {
+    if (a1) {
```

Comment:
Parameter looks like its BOOL and probably called "sell" or somethin like that?

Reply:
Nope its a int, BuildingClass::Sell_Back checks it for -1, 0, 1.

Reply:
Fair enough.